resources/js/app.jsx

```diff
@@ -1,7 +1,9 @@
 import './bootstrap';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import HelloReact from './components/HelloReact';
+import HelloChild from './components/HelloChild';
 
 const root = ReactDOM.createRoot(document.getElementById('react-root'));
-root.render(<HelloReact />);
+root.render(<React.StrictMode>
+                <HelloChild name="Akihiro" />
+            </React.StrictMode>);
```

Comment:
インデントがずれているので修正お願いいたします
```suggestion
root.render(
            <React.StrictMode>
                <HelloChild name="Akihiro" />
            </React.StrictMode>);
```